GSoC_resources.md

```diff
@@ -6,67 +6,65 @@ permalink: gsoc_resources
 All of us go through the same process of devouring the internet when we are writing our proposal for GSoC. This is an attempt to bookmark all those resources in one place.
 
-## GSoC 2017 Ideas page
+Please fill this [GSoC accepted proposal form](https://forms.gle/xaoXmpEdeAJ9c9Fn9), so that it could help other potential GSoC candidates to write a more competent proposal.
 
+## This year selected proposals
+### GSoC 2019
 
-* [Gnome](https://wiki.gnome.org/Outreach/SummerOfCode/2017/Ideas)
+* [Abhishek Singh/ESIP](https://summerofcode.withgoogle.com/projects/#4567128563253248)
+* [Mahesh Tudu @ FreeBSD](https://summerofcode.withgoogle.com/projects/#6555942194249728)
+* [Romit Kumar @ BRL-CAD](https://summerofcode.withgoogle.com/projects/#5042363104231424)
+* [Jayjeet Chakraborty @ CROSS](https://summerofcode.withgoogle.com/projects/#6075586542305280)
+* [Avinash Aggarwal/SugarLabs](https://summerofcode.withgoogle.com/projects/#5993905961566208)
+* [Ayan Banerjee @ UCSC Xena](https://summerofcode.withgoogle.com/projects/#5795682349219840)
+* [Sudipta Pandit @ The Honeynet Project](https://summerofcode.withgoogle.com/projects/#6189934577188864)
 
-* [libvirt - virtualization API](https://wiki.libvirt.org/page/Google_Summer_of_Code_Ideas)
-
-* [LibreOffice](https://wiki.documentfoundation.org/Development/GSoC)
-
-* [KDE](https://community.kde.org/GSoC/2017/Ideas)
-
-* [AppImage project](http://discourse.appimage.org/t/google-summer-of-code-2017/106)
-
-* [JBoss](https://developer.jboss.org/wiki/JBossCommunityGoogleSummerOfCode2017Ideas)
-
-* [CERN](https://ep-dep-sft.web.cern.ch/article/221195)
-
-* [Idris](https://github.com/idris-lang/Idris-dev/wiki/Google-Summer-of-Code-2017-ideas-&-plans)
-
-* [GNU](https://www.gnu.org/software/soc-projects/ideas-2017.html)
-
-* [BeagleBoard](http://elinux.org/BeagleBoard/GSoC/Ideas)
-
-* [Eclipse](https://wiki.eclipse.org/Google_Summer_of_Code_2017_Ideas)
-
-* [OSGeo](https://wiki.osgeo.org/wiki/Google_Summer_of_Code_2017_Ideas)
-
-* [redhenlab](http://www.redhenlab.org/summer-of-code/summer-of-code-2017---ideas-page)
-
-* [Ruby](https://github.com/rubygsoc/rubygsoc/wiki/Ideas-List)
+## Previous years selected proposals
 
-* [PublicLab](https://publiclab.org/wiki/gsoc-ideas)
+#### GSoc 2018
 
-* [Coala](http://projects.coala.io/)
+* [Ankan Poddar 2nd year CS/CROSS](https://summerofcode.withgoogle.com/archive/2018/projects/5223266494971904/)
+* [Anirban Banik/PocketBeagle](https://summerofcode.withgoogle.com/archive/2018/projects/5031706255949824/)
+* Sudhanshu Saurav
+* [Lokesh Nandanwar/So-Sy Labs](https://summerofcode.withgoogle.com/archive/2018/projects/4726926518255616/)
+* [Akshat Garg/FOSSASIA](https://summerofcode.withgoogle.com/archive/2018/projects/4537155267330048/)
+* [Hrishav Mukherjee/Boston University](https://summerofcode.withgoogle.com/archive/2018/projects/5687777542799360/)
 
-* [FOSSASIA](http://labs.fossasia.org/ideas.html)
+#### GSoc 2017
 
-* [OpenSUSE](http://101.opensuse.org/)
+* Naveen
```

Comment:
[Naveen Kumar Sangi](https://summerofcode.withgoogle.com/archive/2017/projects/4690491102724096/)

Reply:
Don't ignore changes and ask for a new review, please. @romitkarmakar

Reply:
Added this link.